language/evm/stdlib/sources/ExternalResult.move

```diff
@@ -0,0 +1,105 @@
+/// This module defines the ExternalResult type and its methods
+/// ExternalResult is used to receive the return value from an external call (https://docs.soliditylang.org/en/v0.8.13/control-structures.html?highlight=try#try-catch)
+module Evm::ExternalResult {
+    use Std::Option::{Self, Option};
+    use Evm::U256::U256;
+
+    /// This struct will contain either a value of type T or an error
+    /// value stores return value when the execution succeeds
+    /// err_str stores the message as vector<u8> if the error was caused by revert("reasonString") or require(false, "reasonString")
+    /// panic_code stores the error code if the error was caused by a panic
+    /// err_vec stores the error information as dynamic byte array (vector<u8>)
+    struct ExternalResult<T> has copy, drop, store {
+        value: Option<T>,
+        err_vec: Option<vector<u8>>,
```

Comment:
One suggestion. Could we name this as `err_data`, and comment that it represent the low level (raw) data?